### PR TITLE
HEEDLS-483 - Home and Iframe accessibility bug fixes.

### PR DESCRIPTION
**Pull request by DanBloxham-sw**
Updated Learning Content iframes to use new titles, with new breadcrumb. 
Changes iframes to have a min-height of 500px on desktops so they are a little more useable. Mobile has been left as it is to minimise confusion on the reliance of two scrollers.
Other minor changes also performed.

Ran all unit tests,
Tested in chrome,
Checked with screenreader

Example screen shot of learning content page:

image

### CODE REVIEW

File: DigitalLearningSolutions.Web/ViewModels/LearningContent/LearningContentViewModel.cs [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-        public readonly string Brand;
-
-        public LearningContentViewModel(string brand)
-        {
-            Brand = brand;
-        }
-    }
-}
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningContent
+{
```
Comment (DanBloxham-sw) [inline DigitalLearningSolutions.Web/ViewModels/LearningContent/LearningContentViewModel.cs:2]: To clarify, this is definitely now CRLF line endings

File: DigitalLearningSolutions.Web/Views/FindYourCentre/Index.cshtml [hunk 2]
```diff
@@ -1,10 +1,15 @@
-﻿@using DigitalLearningSolutions.Web.Helpers
-@using DigitalLearningSolutions.Web.ViewModels.FindYourCentre
+﻿@using DigitalLearningSolutions.Web.ViewModels.FindYourCentre
 @model FindYourCentreViewModel
 @{
   ViewData["Title"] = "Find Your Centre";
```
Comment (AlexJacksonDS) [inline DigitalLearningSolutions.Web/Views/FindYourCentre/Index.cshtml:4]: Should this be capitalised like this? Doesn't match the heading capitalisation

**Review by AlexJacksonDS — COMMENTED** [review]
Just one question about capitalisation

File: DigitalLearningSolutions.Web/Views/FindYourCentre/Index.cshtml (hunk 2 above)
Comment (DanBloxham-sw) [reply DigitalLearningSolutions.Web/Views/FindYourCentre/Index.cshtml:4]: According to @stellake , the capitalisation here is correct, but the consistency isn't too important. 

"So H1 should be only the first letter capitalised
But titles are opposite and usually have every first letter capitalised
But I wouldn't worry too much about it, the titles are only in the tab and mainly for screenreader users"

Comment (AlexJacksonDS) [reply DigitalLearningSolutions.Web/Views/FindYourCentre/Index.cshtml:4]: OK so we're just inconsistent with uncapitalised titles on half the pages then, but this one is correct.

**Review by AlexJacksonDS — APPROVED** [review]

**Review by stellake — APPROVED** [review]
Looks good 👍

**Pull request merged**